src/generators/static/readmeFileGenerator.ts

```diff
@@ -27,10 +27,12 @@ npm install ${packageDetails.name}
 \`\`\`
 
 ### How to use
+#### Authentication
+One of the first steps in accessing the services using SDK is to authenticate the client. There are several methods to authenticate such as interactive login, etc. Refer [readme for @azure/ms-rest-nodeauth](https://www.npmjs.com/package/@azure/ms-rest-nodeauth) package for all options to authenticate the client.
```

Comment:
This PR is against the v6 branch.

From my understanding the packages generated using v6 of the code generator use `TokenCredential` and therefore a user will have to bring in `@azure/identity`. Here, we are talking about `@azure/ms-rest-nodeauth` which is for the packages generated using v4 of the code generator.

Reply:
The issue https://github.com/Azure/autorest.typescript/issues/721 is mainly for packages generated using the v4 code generator. The packages generated using the v6 generator are getting the convenience layer and hand written readmes anyway and so does not have this immediate need.

Reply:
@ramya-rao-a  Does the below line makes sense? 

```
One of the first steps in accessing the services using SDK is to authenticate the client. There are several methods to authenticate such as DefaultAzureCredential, etc. Refer [readme for @azure/identity](https://www.npmjs.com/package/@azure/identity) package for all options to authenticate the client.
```